openmdao/drivers/test/test_pyoptsparse.py

```diff
@@ -131,8 +130,8 @@ def test_simple_paraboloid_equality(self):
         prob.run()
 
         # Minimum should be at (7.166667, -7.833334)
-        assert_rel_error(self, prob['x'], 7.16667, 1e-6)
-        assert_rel_error(self, prob['y'], -7.833334, 1e-6)
+        assert_rel_error(self, prob['x'], 7.16667, 1e-3)
```

Comment:
yes.. with ACC at 1e-9, I can set the test precision back to 1e-6... I'll push that up